public/404/index.html

```diff
@@ -0,0 +1,509 @@
+                  class="nav-link"
+                  href="https://www.meetup.com/Official-Metronome-Meetup/"
+                  target="_blank"
+                  ><i class="fab fa-meetup"></i> Meetup</a
+                >
+              </li>
+            </ul>
+          </div>
+          <div class="col-sm-6 col-lg-3">
+            <h5>CONNECT</h5>
+            <p class="mb-3">
+              Sign up to receive the latest updates from the Metronome team:
+            </p>
+            <div class="sendgrid">
+              <script
+                src="//sgwidget.leaderapps.co/js/sendgrid-widget.js"
+                type="text/javascript"
+              ></script>
+              <style>
+                .sendgrid-subscription-widget .response {
+                  display: none;
+                  font-family: sans-serif;
+                  font-size: 0.8em;
+                }
+                .sendgrid-subscription-widget .success {
+                  color: #7e61f8;
+                  text-align: center;
+                }
+                .sendgrid-subscription-widget .error {
+                  color: red;
+                }
+              </style>
+              <div
+                class="sendgrid-subscription-widget"
+                id="sendgrid-subscription-widget"
+              >
+                <form
+                  data-token="ad8b1f4cd2963a816efc51879519201e"
+                  id="sg-widget"
+                  name="sg-widget"
+                  onsubmit="return false;"
+                >
+                  <input
+                    class="email form-control mb-1"
+                    id="sg_email"
+                    name="sg_email"
+                    placeholder="Email Address"
+                    required=""
+                    type="email"
+                  />
+                  <input
+                    class="btn btn-primary font-weight-bold w-100"
+                    id="sg-submit-btn"
+                    type="submit"
+                    value="SIGN UP FOR UPDATES &raquo;"
+                  />
+                  <div class="sg-response" id="sg-response"></div>
+                </form>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="row copyright mt-5 ml-n1 mr-n1">
+          <div class="col-md-9 mb-5 pl-0">
+            <p>
+              Copyright &copy; 2018-2019 Metronome. All Rights Reserved. Site
+              Design by
+              <a href="https://thegrove.co" target="_blank">The Grove</a>
+            </p>
+          </div>
+          <div class="col-md-3 mb-5 pr-0 text-right">
+            <a href="/privacy/">Privacy Policy</a>
+          </div>
+        </div>
+      </div>
+    </footer>
+    <!-- /FOOTER -->
+    <!-- SCRIPTS -->
+    <script src="../dist/js/main.js?cb=1570034155144"></script>
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=%3C%-htmlWebpackPlugin.options.gaTrackingId%%3E"
```

Comment:
Fix template delimiters: `%3C%` to `<%` and `%%3E` to `%>` to allow proper injection of Google tracking ID.

Reply:
All set, just pushed that change.  

Let me know if you need anything else.